### DISCUSSION TITLE
Data stream not visible.

### DISCUSSION THREAD

**Discussion by sahejeventme** [invertase/react-native-firebase#6755] [Q&A]
I've integrated react native firebase analytics with custom events. I see data being present in GA, also in debug view I'm able to view all the events triggered. I'm unable to view the data in data stream.
<img width="367" alt="image" src="https://user-images.githubusercontent.com/115151158/205034949-3cebeebe-7a11-42f9-894f-658ff51a8c99.png">
I get No data received in past 48 hrs. Is there any other configuration we need to do to view data in data stream?

**Comment by mikehardy** [comment]
> I see data being present in GA, also in debug view I'm able to view all the events triggered

If that's the case, that means that events are sent from device to google analytics, stated differently: react-native-firebase is doing it's job of recording events and transferring them to google

I don't think there's anything more to do with regard to react-native-firebase. As a general firebase question then, you might post on stackoverflow, firebase offers official support there
